src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -282,19 +283,19 @@ import TabelaCustomizada from './TabelaCustomizada.astro';
 </TabelaCustomizada>
 ```
 
-Note que slots nomeados precisam ser um filho imediato de um componente. Você não pode passar slots nomeados através de elementos aninhados.
+Note que slots nomeados precisam ser um filho imediato do componente. Você não pode passar slots nomeados através de elementos aninhados.
 
 :::tip
 Slots nomeados também podem ser passados para [componentes de frameworks de UI](/pt-br/guides/framework-components/)!
 :::
 
 :::note
-Não é possível gerar dinamicamente um slot nomeado no Astro, como por exemplo dentro de uma função map. Se essa funcionalidade é necessária dentro de um componente de um framework de UI, o melhor é gerar esses slots dinâmicos dentro do próprio framework.
+Não é possível gerar dinamicamente o nome de um slot no Astro, como por exemplo dentro de uma função map. Se essa funcionalidade é necessária dentro de um componente de um framework de UI, o melhor é gerar esses slots dinâmicos dentro do próprio framework.
```

Comment:
Translation says it's not possible to dynamically generate an "Astro slot name", referring to the name rather than the slot.